pages/common/starship.md

```diff
@@ -0,0 +1,36 @@
+# starship
+
+> The minimal, blazing-fast, and infinitely customizable prompt for any shell.
+> More information: <https://starship.rs>.
+
+- Print the shell function used to execute starship:
+
+`starship init {{bash|elvish|fish|ion|powershell|tcsh|zsh}}`
```

Comment:
Isn't the shell function to execute starship `starship`?

Reply:
Yep, but the installation is different for every shell.
```
eval "$(starship init bash)"
Invoke-Expression (&starship init powershell)
eval (starship init elvish)
eval `starship init tcsh`
```

Maybe I can create a subcommand page for `starship-init` similar to `gh-completion`

Reply:
Not a bad idea, but that's for another PR.